pkg/invoke/args.go

```diff
@@ -19,6 +19,7 @@ import (
 	"strings"
 )
 
+// CNIArgs represents extra arguments passed in by the user at invocation time
```

Comment:
`CNIArgs` represents the arguments to be passed to the plugin via the process environment. Sometimes these must be assembled from configuration -- see the use just below here -- and sometimes (when calling a plugin recursively) they must be inherited from the calling process.